ui/src/app/users/asideForm.tsx

```diff
@@ -4,29 +4,27 @@ import { changePassword, postUser } from "../queries";
 import { ChangeEvent, useContext, useState } from "react";
 import { AsideContext } from "../aside";
 import { useAuth } from "../auth/authContext";
+import { Panel, Button, Input, PasswordToggle, Form } from "@canonical/react-components";
 
 export default function UsersPageAsidePanel() {
     const asideContext = useContext(AsideContext)
+    const panelTitle = asideContext.extraData == null ? "Add a New User" : "Change User Password"
```

Comment:
This thing was and still is pretty awkward and we'd save a bunch of time in the future if we refactored it before adding new tabs to notary.

Reply:
agreed, navigation and asides still have to be done, but I'd do those in a separate pr as this one is large enough